com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/HDDynamicShadowAtlas.cs

```diff
@@ -172,53 +172,51 @@ public void AddRequestToPendingBlitFromCache(HDShadowRequest request)
                 m_MixedRequestsPendingBlits.Add(request);
         }
 
-        public void ClearPendingBlitsRequests()
```

Comment:
Why removing these? I liked them being explicit functions

Reply:
Oh I see because the clear is inside the render function now so cannot get reasonable info before rg execution; fair enough